public/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+	<meta charset="utf-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+	<meta name="theme-color" content="#fff">
+	<title>Download MyCrypto</title>
+</head>
+
+<body>
+	<noscript>
+		You need to enable JavaScript to run this app.
```

Comment:
I'm really not a fan of requiring Javascript for such a simple page.

Reply:
Maybe we should look into server side rendering, and making that a part of the compile process.